.github/workflows/terraform-integration-tests.yml

```diff
@@ -56,7 +56,7 @@ jobs:
           if [[ $hz_version == *"SNAPSHOT"* ]]; then
             wo_SNAPSHOT=$( echo $hz_version | grep -Po "[.a-zA-Z0-9]+(?=-SNAPSHOT)" )
             snapshot_link=$(curl -s "https://oss.sonatype.org/content/repositories/snapshots/com/hazelcast/hazelcast/$hz_version/" 2>&1 \
-                 | grep -Po  "hazelcast-$wo_SNAPSHOT-\d{8}.\d{6}-\d+\.jar" | head -1)
+                 | grep -Po  "hazelcast-$wo_SNAPSHOT-\d{8}.\d{6}-\d+\.jar" | tail -1)
```

Comment:
Can you please explain in detail why this is necessary?

Reply:
I am using maven now, this is outdated.

Reply:
Thanks. Nice to see maven way applied.